gatling-core/src/main/resources/gatling-defaults.conf

```diff
@@ -43,6 +43,7 @@ gatling {
   charting {
     noReports = false       # When set to true, don't generate HTML reports
     maxPlotPerSeries = 1000 # Number of points per graph in Gatling reports
+    useGroupDurationInsteadOfCumulatedResponseTime = false  # Switch group timings from cumulated response time to group duration.
```

Comment:
It's a switch between two kind of timings for groups. 
Firstly, I wanted to let the possibility to display all timings in report like this : 
mode 1 : cumulatedResponseTime
mode 2 : groupDuration
mode 3 : cumulatedResponseTime | groupDuration

It's more complex to implement, but we could begin by implementing mode 1 and 2. And use a switch paramater name like : groupTimingsDisplayMode = 1